specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/examples/Volumes_PreRestore_MaximumSet_Gen.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "Volumes_PreRestore_MaximumSet",
+  "operationId": "Volumes_PreRestore",
+  "parameters": {
+    "subscriptionId": "67B407F8-ED08-4144-B074-BA08FF4FB978",
```

Comment:
Please hide the subscriptionId value, don't use a read subscriptionId in example, include in the resource ID of "diskSnapshotIds" value.
You can refer other swagger examples.